WorkflowUI/Sources/Screen/ScreenViewController.swift

```diff
@@ -44,6 +44,7 @@ open class ScreenViewController<ScreenType: Screen>: WorkflowUIViewController {
         ScreenType.self
     }
 
+    @_alwaysEmitIntoClient
```

Comment:
It'd be great to add some context around this so that future-us will know what the heck this was added for.

Reply:
also... is this a workaround for a linker issue? can we deal with it via build args/configuration vs changing the source? is this a bug that should be reported somewhere?

Reply:
> is this a workaround for a linker issue? 

This is a linker issue; however, it works fine in SPM land. There's something about the Square Bazel build infrastructure (and the current versions we're pinned at) which results in this.

> can we deal with it via build args/configuration vs changing the source? 

@congt tried a ton of things, and this was the only thing we could do to solve it previously 

> is this a bug that should be reported somewhere?

Hard to say since it's Bazel specific and possibly Square's rules specific